mr/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergOutputCommitter.java

```diff
@@ -188,7 +193,13 @@ private JobConf jobConf(Table table, int taskNum) {
     conf.setNumReduceTasks(0);
     conf.set(HiveConf.ConfVars.HIVEQUERYID.varname, QUERY_ID);
 
-    HiveIcebergStorageHandler.put(conf, table);
+    Map<String, String> propMap = new HashMap<>();
```

Comment:
Is this needed because a value may be `null`? If not, we prefer to use `Maps.newHashMap` that enforces non-null keys and values.

Reply:
Fixed